napari/components/dims.py

```diff
@@ -184,10 +184,7 @@ def not_displayed(self) -> Tuple[int, ...]:
 
     @property
     def displayed_order(self) -> Tuple[int, ...]:
-        displayed = self.displayed
-        # equivalent to: order = np.argsort(self.displayed)
-        order = sorted(range(len(displayed)), key=lambda x: displayed[x])
-        return tuple(order)
+        return argsort(self.displayed)
```

Comment:
@brisvag : I think you made a comment on the lack of test coverage for this property. It's probably because we don't use it anywhere internally in napari. So I added to an existing test in `test_dims` to cover these three related properties.